CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Change Log
 
+## Version 1.41.1
+* Release date: January 30, 2023
+* Release status: General Availability
+
+### Bug fixes in 1.41.1
+
+| New Item | Details |
+| --- | --- |
+| Connection | Fixed a regression causing Object Explorer to not show database objects for Azure SQL DB Basic SLO |
+| Object Explorer | Fixed a bug causing incorrect Azure account tenant selection when connecting to server through Azure view |
```

Comment:
I think you need to toggle areas for both issues.

Reply:
oops, thanks!